api/system-dhcp/validate

```diff
@@ -51,6 +51,13 @@ if ($action == 'create-reservation') {
         $v->addValidationError('IpAddress','dhcp_reservation_system_validator');
     }
 
+    # valid if the mac address is not already reserved
```

Comment:
This new code shouldn't be needed since the same validation rule is already present inside the 'dhcp-reservation' system validator called above.